build/Helix/GenerateTestProjFile.ps1

```diff
@@ -278,7 +292,7 @@ foreach ($testModule in $testModules)
 
     <HelixWorkItem Include="$($testClass.Name)" Condition="'`$(TestSuite)'=='$($JobTestSuiteName)'">
       <Timeout>00:20:00</Timeout>
-      <Command>call %HELIX_CORRELATION_PAYLOAD%\runtests.cmd /select:"(@Name='$($testClass.Name).*'$(if ($testSuiteExists) { "and not @TestSuite='*'" }))"</Command>
+      <Command>call %HELIX_CORRELATION_PAYLOAD%\runtests.cmd /select:"(@Name='$($testClass.Name).*'$(if ($testSuiteExists) { "and not @TestSuite='*'" }))$($TaefQueryToAppend)"</Command>
```

Comment:
Is this needed?  We should've already filtered out tests we don't want by passing $TaefSelectQuery into the /listproperties call.

Reply:
Yeah, it is still needed. If we want to run a specific ColorPicker test, /listproperties will generate the filtered list, but the proj would still generate a WorkItem to run all ColorPicker tests.